tests/test_matprod.py

```diff
@@ -1,7 +1,15 @@
 """Tests that the various matprod methods produce the same result."""
 
+import sys
+
+sys.path.insert(
+    0,
+    "/home/lab-admin/Desktop/Desktop/Graduate_School/Research/ASU/21_group/matvis/src/",
+)
+
```

Comment:
This needs to be removed. You might want to install at the top level with `pip install -e .` so that when running `pytest` it can find the library.

Reply:
My bad on the mess with this file.  I had meant to replace this with a cleaned up version and evidently forgot to.